python/py-cryptography/Portfile

```diff
@@ -32,18 +33,93 @@ platform macosx {
         patchfiles-append   patch-src__cffi_src_build_openssl.py.diff
     }
 }
+if {${name} ne ${subport}
+    && [string match "py*-${github.project}" ${subport}]} {
+    depends_build-append \
+                    port:py${python.version}-setuptools-rust \
```

Comment:
this only has subports for py37/38/39

Reply:
That’s what upstream supports, ignoring EOL 36:

https://github.com/PyO3/setuptools-rust/blob/61b0603ba8cecaff34331904de8f2a0d035152ca/setup.cfg#L31

Reply:
sure, but that means that you will need to remove the PY27/PY35 subports from `py-cryptography` (which doesn't support lower versions either) and add PY36 to `py-setuptools-rust`

Reply:
and Python 3.6 is not EOL [yet](https://devguide.python.org/#status-of-python-branches)

Reply:
Right, not dead yet. I kept it in the latest push.